install

```diff
@@ -305,6 +305,10 @@ else
   puts "Install #{Tty.white}Xcode#{Tty.reset}: https://developer.apple.com/xcode" unless File.exist? "/usr/bin/cc"
 end
 
+puts "Add to your #{Tty.white}.bashrc#{Tty.reset} or #{Tty.white}.zshrc#{Tty.reset}:"
+puts '\texport PATH="$HOME/.linuxbrew/bin:$PATH"'
```

Comment:
The `\t`'s aren't getting converted to tabs. This is going to take a bit of work to find output that does not look terrible.